axelar-chains-config/info/devnet-amplifier.json

```
@@ -1,7 +1,7 @@
 {
   "chains": {
     "avalanche": {
-      "name": "Avalanche",
+      "name": "avalanche",
```

```suggestion
      "name": "Avalanche",
```
This is the display name, doesn't need to be lower cased. `axelarId` field should be used instead for identifiers